bin/carthage-bootstrap-if-needed

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+if ! diff Cartfile.resolved Carthage/Cartfile.resolved &>/dev/null; then
+  carthage bootstrap 
+  cp Cartfile.resolved Carthage
+fi
```

Comment:
I see. I was using git submodules directly on CI but I've been installing Carthage just so I can use the same script as locally.